Packs/BreachRx/Integrations/BreachRx/BreachRx.py

```diff
@@ -0,0 +1,321 @@
+            )
+
+        incidents = [client.get_incident(incident_name, incident_identifier)]
+
+        if not incidents:
+            raise Exception("Error: No BreachRx privacy Incident found using the search terms provided.")
+
+    if type(incidents) is not list:
+        incidents = [incidents]
+
+    for incident in incidents:
+        incident["actions"] = client.get_actions_for_incident(incident['id'])
+
+        for action in incident["actions"]:
+            action["phase_name"] = action["phase"]["name"]
+
+    readable_output = ""
+
+    for incident in incidents:
+        actions_markdown_table = tableToMarkdown("Actions", incident["actions"], headers=["name", "phase_name"])
+        readable_output += f"# {incident['name']} ({incident['id']})\n" + actions_markdown_table + "\n"
+
+    return CommandResults(
+        outputs_prefix="BreachRx.Incident",
+        outputs_key_field="id",
+        outputs=incidents,
+        raw_response=incidents,
+        readable_output=readable_output
+    )
+
+
+def import_incident_command(
+    client: BreachRxClient,
+    incident_name: str = None,
+    incident_identifier: str = None
+) -> Union[CommandResults, str]:
+    incident = client.get_incident(incident_name, incident_identifier)
+
+    if not incident:
+        raise Exception("Error: No BreachRx privacy Incident found using the search terms provided.")
+
+    return CommandResults(
+        outputs_prefix="BreachRx.Incident",
+        outputs_key_field="id",
+        outputs=incident,
+        raw_response=incident,
+        readable_output=f"Incident imported with name={incident.get('name')}."
+    )
+
+
+def get_incident_command(
+    client: BreachRxClient,
+    incident_name: str = None,
+    incident_identifier: str = None
+) -> Union[CommandResults, str]:
+    incident = client.get_incident(incident_name, incident_identifier)
+
+    if incident:
+        return CommandResults(
+            raw_response=incident,
+            readable_output=f'Incident found with name="{incident.get("name")}" and identifier="{incident.get("identifier")}".'
+        )
+    else:
+        return "No Incident found with those search terms."
+
+
+COMMANDS = {
+    "test-module": test_module,
+    "breachrx-incident-create": create_incident_command,
+    "breachrx-incident-actions-get": get_incident_actions_command,
+    "breachrx-incident-import": import_incident_command,
+    "breachrx-incident-get": get_incident_command,
+}
+
+
+def is_valid_url(url):
+    try:
+        result = urlparse(url)
+        return all([result.scheme, result.netloc])
+    except ValueError:
+        return False
+
+
+def main() -> None:  # pragma: no cover
+    try:
+        base_url = demisto.params()["api_url"]
+        org_name = demisto.params()["url"].split(".")[0].replace("https://", "")
+        api_key = demisto.params().get("api_key", {}).get("password")
+        secret_key = demisto.params().get("secret_key", {}).get("password")
```

Comment:
If the `demisto.params().get("api_key", {})` results in `{}` then there will not be a `password` attribute to the (empty) dictionary and an `AttributeError` will be raised.

This can be seen if we populate `params` method in `demistomock` with dummy attributes:

```python
def params():
    """(Integration only)
    Retrieves the integration parameters object

    Returns:
      dict: Integrations parameters object

    """
    return {
      "url": "https://example.app.breachrx.io",
      "api_url": "https://graphql.app.breachrx.io",
      "secret_key": "1234",
      "api_key": "5678"
      }
```

```bash
❯ python BreachRx.py
Traceback (most recent call last):
  File "BreachRx.py", line 297, in main
    api_key = demisto.params().get("api_key", {}).get("password")
AttributeError: 'str' object has no attribute 'get'
```

To fix, we can do something like this:

```suggestion
        api_key = demisto.params().get("api_key", {}).get("password") if "password" in demisto.params().get("api_key") else ""
        secret_key = demisto.params().get("secret_key", {}).get("password")
```